ESP8266Interface.h

```diff
@@ -264,6 +280,7 @@ class ESP8266Interface : public NetworkStack, public WiFiInterface
     bool _ids[ESP8266_SOCKET_COUNT];
     int _initialized;
     int _started;
+    int _keepalives[ESP8266_SOCKET_COUNT];
```

Comment:
Again can't put this in the socket structure?

Reply:
Will do it